R/copy.R

```diff
@@ -64,14 +64,14 @@ file_copy <- function(path, new_path, overwrite = FALSE) {
 
 #' @rdname copy
 #' @export
-dir_copy <- function(path, new_path) {
+dir_copy <- function(path, new_path, overwrite = FALSE) {
   assert_no_missing(path)
   assert_no_missing(new_path)
   assert("`path` must be a directory", all(is_dir(path)))
   assert("Length of `path` must equal length of `new_path`", length(path) == length(new_path))
 
   for (i in seq_along(path)) {
-    if (isTRUE(unname(is_dir(new_path[[i]])))) {
+    if (isFALSE(overwrite) & isTRUE(unname(is_dir(new_path[[i]])))) {
```

Comment:
`isFALSE` is only available in R 3.5+, so we would need to use `!isTRUE()`. Also because these are scalar logicals we should use `&&` instead of the vectorized `&`. I will make these changes.